### PR TITLE
refactor(api-client): Limit valid user accent colors

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: packages/api-client/src/main/user/UserAccentColor.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+export type UserAccentColor = 1 | 2 | 3 | 4 | 5 | 6 | 7;
```
Comment (ffflorian) [inline packages/api-client/src/main/user/UserAccentColor.ts]:
```suggestion

export type UserAccentColor = 1 | 2 | 3 | 4 | 5 | 6 | 7;
```

**Comment by ffflorian** [comment]
Best version ever :tada:

File: packages/commons/src/main/util/AccentColor.ts [hunk 2]
```diff
@@ -19,51 +19,62 @@
 
 import * as RandomUtil from './RandomUtil';
 
+export enum UserAccentColor {
```
Comment (Yserz) [inline packages/commons/src/main/util/AccentColor.ts]: Naming suggestion `AccentColorID`

File: packages/api-client/src/main/user/UserAccentColor.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+export enum UserAccentColor {
```
Comment (Yserz) [inline packages/api-client/src/main/user/UserAccentColor.ts]: That can be removed now, right?

File: packages/api-client/src/main/auth/RegisterData.ts [hunk 4]
```diff
@@ -17,11 +17,12 @@
  *
  */
 
+import {UserAccentColor} from '@wireapp/commons/dist/commonjs/util/AccentColor';
```
Comment (Yserz) [inline packages/api-client/src/main/auth/RegisterData.ts]: Is it possible to have an import in the form of `import {Something} from '@wireapp/commons` here?

Comment (bennyn) [reply packages/api-client/src/main/auth/RegisterData.ts]: How do you import `AccentColor` at the moment in other projects?

**Pull request merged**